components/chat_widget/src/components/ocs-chat/ocs-chat.tsx

```diff
@@ -401,47 +523,23 @@ export class OcsChat {
       <Host>
         <button class="btn" onClick={() => this.load()}>{this.buttonText}</button>
         {this.visible && (
-          <div id="ocs-chat-window" class={this.getPositionClasses()}>
+          <div
+            ref={(el) => this.chatWindowRef = el}
+            id="ocs-chat-window"
+            class={this.getPositionClasses()}
+            style={this.getPositionStyles()}
+          >
             {/* Header */}
-            <div class="flex justify-between items-center px-2 py-2 border-b border-gray-100">
-              <div class="flex gap-1">
-                <button
-                  class={{
-                    'hidden sm:block p-1.5 rounded-md transition-colors duration-200 hover:bg-gray-100': true,
-                    'text-blue-600': this.position === 'left',
-                    'text-gray-500': this.position !== 'left'
-                  }}
-                  onClick={() => this.setPosition('left')}
-                  aria-label="Dock to left"
-                  title="Dock to left"
-                >
-                  <ArrowLeftEndOnRectangleIcon/>
-                </button>
-                <button
-                  class={{
-                    'p-1.5 rounded-md transition-colors duration-200 hover:bg-gray-100': true,
-                    'text-blue-600': this.position === 'center',
-                    'text-gray-500': this.position !== 'center'
-                  }}
-                  onClick={() => this.setPosition('center')}
-                  aria-label="Center"
-                  title="Center"
-                >
-                  <ViewfinderCircleIcon/>
-                </button>
-                <button
-                  class={{
-                    'p-1.5 rounded-md transition-colors duration-200 hover:bg-gray-100': true,
-                    'text-blue-600': this.position === 'right',
-                    'text-gray-500': this.position !== 'right'
-                  }}
-                  onClick={() => this.setPosition('right')}
-                  aria-label="Dock to right"
-                  title="Dock to right"
-                >
-                  <span class="hidden sm:block"><ArrowRightEndOnRectangleIcon/></span>
-                  <span class="sm:hidden"><ArrowDownOnSquareIcon/></span>
-                </button>
+            <div
+              class={`flex justify-between items-center px-2 py-2 border-b border-gray-100 sm:${this.isDragging ? 'cursor-grabbing' : 'cursor-grab'} active:bg-gray-50 sm:hover:bg-gray-25 transition-colors duration-150`}
+              onMouseDown={this.handleMouseDown}
+              onTouchStart={this.handleTouchStart}
+            >
+              {/* Drag indicator */}
+              <div class="hidden sm:flex gap-1">
+                <div class="flex gap-0.5 ml-2 pointer-events-none">
+                  <EllipsisHorizontalIcon/>
```

Comment:
How about using the [grid dots icon](https://fontawesome.com/icons/grip-dots-vertical?s=solid) instead? I've seen ellipsis generally being used for "more options" type of operations